### ISSUE TITLE
Improve the README

### ISSUE THREAD

**Issue by noviluni** [scrapinghub/dateparser#723]
This library doesn't have a website (apart from the docs), so the README should be treated as a presentation of the library.

I think we could add a fresher vision over this library and focus it over the use cases instead of including parts from the docs.

- [x] Add new logo ~(blocked by https://github.com/scrapinghub/dateparser/pull/719)~
- [x] Installation
- [x] Supported Python versions
- [x] Basic examples (current content with some simplifications) with basic settings.
- [x] Common use cases (more on this below)
- [ ] Multiple links to docs (how to contribute, etc).
- [x] Recommendation: to use `languages` or `locales` when knowing the language to improve performance and unexpected behaviors.
- [x] Explaining that what you have to pass to the `parse()` method is a date string, not a string containing a date.
- [ ] Explain that `search_dates` is not fully supported and it has a lot of limitations.

If you have any idea don't hesitate to add a comment!

-----
#### When consuming data from different sources:
* Scraping: extract dates from different places with several different formats and languages
* IoT: consuming data coming from different sources with different date formats
* Tooling: consuming dates from different logs / sources
* Format transformations: when transforming dates coming from files (PDF, CSV, etc.) to other formats (database, etc).

#### When offering natural interaction with users
* Tooling and CLI: allow users to write “3 days ago” to retrieve information.
* Search engine: allow people to search by date in an easiest / natural format.
* Bots: allow users to interact with a bot easily

**Comment by noviluni** [comment]
I started working on this.

My idea is to create a new README.md file (which is better supported by Github) and move the current content or part of the content to the docs. You can see a preview of the version I'm working here: https://github.com/noviluni/dateparser/tree/improve_readme

@Gallaecio any feedback will be welcomed :)

**Comment by noviluni** [comment]
Added new PR! #779

**Comment by noviluni** [comment]
As the PR #779 has been merged I will close this issue. 

The missing sections will be omitted from the README as they are in the docs (and the `search_dates` has its own ticket: https://github.com/scrapinghub/dateparser/issues/737).
